core/version_comparator.go

```diff
@@ -0,0 +1,86 @@
+package core
+
+import (
+	"regexp"
+
+	"github.com/pkg/errors"
+)
+
+// go does not support lookaheads or lookbehinds (tried "^[=,>,<]+[\"d\"-,.]*" "^[=,>,<]+(?=[d])+"
+const OperatorRegex = `^[=><]+`
```

Comment:
This symbol can be unexported because no code outside this file uses this const. I also checked some consuming applications for usage.

It is possible that the same counts for `Operator` as well because methods provide the actual functionality.

- [ ] done